app/src/app/layout/components/feedback-form/feedback-form.component.scss

```diff
@@ -0,0 +1,102 @@
+@import '~src/scss/variables';
+
+.slidercontainer {
+  position: relative;
+  width: 90%;
+  margin: 0 0 0px 5%;
+
+  label {
+    margin: 0;
+  }
+
+}
+
+#feedback-form {
+  width: fit-content;
+  min-width: 1000px;
+  margin: 0 !important;
+}
+
+.no-padding {
```

Comment:
Use existing elements rather than writing generic ones.
For bootstrap utils this will be the same as `.p-0`